### PR TITLE
fix(faucet): use MUD's sendTransaction for better nonce handling

### LINKED ISSUE #2070: faucet should use MUD's sendTransaction
for better nonce handling of lots of drips

### PR DESCRIPTION
**Pull request by R-Morpheus**
closes #2070

### CODE REVIEW

File: packages/faucet/bin/faucet-server.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import fastify from "fastify";
 import { fastifyTRPCPlugin } from "@trpc/server/adapters/fastify";
 import { http, createClient } from "viem";
 import { privateKeyToAccount } from "viem/accounts";
-import { AppRouter, createAppRouter } from "../src/createAppRouter";
+import { AppRouter, createAppRouter } from "../src";
```
Comment (holic) [inline packages/faucet/bin/faucet-server.ts:7]: any reason for this change?

Comment (R-Morpheus) [reply packages/faucet/bin/faucet-server.ts:7]: my ide was swearing and giving out an error. Remove the changes?

Comment (holic) [reply packages/faucet/bin/faucet-server.ts:7]: curious to hear what the error is! but yes, my preference is not using the exports file for local imports

Comment (R-Morpheus) [reply packages/faucet/bin/faucet-server.ts]: my new version of GoLand reacted badly to what could be shortened, had to be fixed

**Pull request merged**